core/trie/trie.go

```diff
@@ -38,14 +38,26 @@ type Trie struct {
 	height  uint
 	rootKey *bitset.BitSet
 	storage Storage
+	hash    func(*felt.Felt, *felt.Felt) *felt.Felt
 }
 
+type NewTrieFunc func(Storage, uint, *bitset.BitSet) *Trie
```

Comment:
I don't see any usage of this. If this is not being used please remove

Reply:
done